src/MoBi.UI/Views/SimulationExplorerView.cs

```diff
@@ -33,10 +38,15 @@ private void compareNodeValues(object sender, CompareNodeValuesEventArgs e)
             e.Result = 0;
 
          //we do not want to sort the items under the simulation node (i.e. no children). Otherwise, Nodes are sorted alphabetically
-         else if (nodeIsSimulationNode(e.Node1.ParentNode))
+         if (nodeIsSimulationNode(e.Node1.ParentNode))
             e.Result = 0;
+         else if(nodeIsModuleConfigurationNode(e.Node1.ParentNode))
+            e.Result = _simulationExplorerPresenter.OrderingComparisonForModules(e.Node1.Tag as ITreeNode<IWithName>, e.Node2.Tag as ITreeNode<IWithName>); 
```

Comment:
I thinl that the casting should be done in the presenter not in the view. 
What is the type of Tag anyways?

Reply:
> What is the type of Tag anyways?

`object`